lib/clAdapters/jira/index.js

```diff
@@ -0,0 +1,66 @@
+import Adapter from '../base/Adapter';
+import url from 'url';
+import request from 'request';
+
+export default class JiraAdapter extends Adapter {
+  constructor() {
+    super();
+    this.apiVersion = 2;
+  }
+
+  init() {
+  }
+}
+
+JiraAdapter.prototype.makeRequest = function(path) {
```

Comment:
Non-blocking: Since you are using the `class` syntax above, would be more consistent to define these methods in the class body (and methods like `init` are already defined there).